src/Psalm/Internal/Analyzer/Statements/ThrowAnalyzer.php

```diff
@@ -84,6 +84,10 @@ public static function analyze(
         }
 
         if ($stmt instanceof PhpParser\Node\Expr\Throw_) {
+            // TODO: Find bug in Psalm:
+            // Conversion to "Type::getEmpty()" triggers Psalm DeprecatedMethod, which is correct
+            // To be investigated: Why isn’t it triggered with "\Psalm\Type::getEmpty()"?!
+            // @phpcs:ignore SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName
```

Comment:
Not a bug. See https://github.com/vimeo/psalm/blob/e1d1ef32b6a7ff15ffd517c29c205d67e8dfe48f/psalm-baseline.xml#L256-L260.

Psalm relies on textual matches for it's baseline feature.

Reply:
Ah, ok. You’ll never stop learning :)

Reply:
so running ``psalm --update-baseline`` should do the trick?!

Reply:
It's `--set-baseline` actually. `--update-baseline` does not add new issues to baseline, it only removes issues that are no longer there.

Reply:
ok…

Reply:
Looks good

Reply:
Arghl… nope. For some reason it removes some lines. I already was wondering why, thougth some old stuff. Local tests run fine…

Reply:
With some cheating… :-)